pkg/kubelet/kubelet.go

```diff
@@ -858,7 +858,7 @@ func (kl *Kubelet) killContainerByID(ID string) error {
 }
 
 const (
-	PodInfraContainerImage = "kubernetes/pause:latest"
+	PodInfraContainerImage = "gcr.io/google_containers/pause:0.8.0"
```

Comment:
Wait...
Not sure what changed since yesterday afternoon, but when I manually docker pull this image, I got message as the following:

```
$ docker pull gcr.io/google_containers/pause:0.8.0
Pulling repository gcr.io/google_containers/pause
2015/04/01 16:28:38 Authentication is required.
```